app/src/main/kotlin/com/github/gotify/NotificationSupport.kt

```diff
@@ -87,6 +205,21 @@ internal object NotificationSupport {
         }
     }
 
+    private fun getChannelID(importance: String, groupId: String): String {
+        return "$importance::$groupId"
+    }
```

Comment:
(nitpick) maybe switch the groupId param with the importance param, as the channels are grouped by groupId.

Reply:
That's a part of the code I've just copied from the previous PR, but I can do that.

Reply:
Resolved in 38bd1b5e32ca38c336435871bca4efd392633e36